src/scss/05_objects/main.scss

```diff
@@ -1,23 +1,17 @@
-.o-row {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
-}
+html {
```

Comment:
Since you're targeting a specific element, this could go inside `elements` or you could even modify your `normalize.css` to include these properties. (Don't be afraid to modify `normalize` – it's generally more performant to customize it to use desired defaults rather than redefining them elsewhere.)

Reply:
Moved to normalize!